docs/source/user_guide/computed_properties.rst

```diff
@@ -147,12 +160,17 @@ You can visualize morphologies using common molecule visualization tools such as
 
     * Click on the `show all frames` button (green circle in the figure below).
 
-    .. image:: computed_properties/morph_viz_jmol.png
-       :alt: morphology visualization with jmol
+
+**Visualisation of** ``structure.mol2`` **with pymol**:
+
+    * In pymol, type `set all_states, 1` to show all molecules.
```

Comment:
Same issue as above, but I again don't mind. The only thing I find confusing in the order for pymol is that you are supposed to enter a command into pymol before you opened it. Should be fixed, my preference would be the old order

Reply:
moved